### ISSUE TITLE
Super Tagger // Display ToC - enhancement

### ISSUE THREAD

**Issue by ghost** [Tizra/Tizra-Customer-Tracker#81]
In the document ToC you have "Display Table of Contents?: Yes/No" as the verbage and options but in super tager you have it setup as "Bypass ToC: Yes/No".

Wouldn't it make more sense if they said and did the same thing?

This is not a bug, just a consistency problem that I have been noticing and finally decided to say something about.

Thanks.
Ryan

**Comment by daviddurand** [comment]
yes, this is a little weird. Internally it' is defined as BypassToC. When I took over development 3 years ago, I made the main editing screen avoid the potential double-negative (_"don't bypass"_) for clarity, but since the supertagger works on all properties, there was no simple way to make exceptions for just some of the properties. I live with myself by thinking that at least the super-tagger and the exports are "advanced features" so that the people who are using them should already at least understand the concepts.

I'll leave this issue open so that it's on the list, but it will probably be there for a while.

**Comment by ghost** [comment]
Makes sense.
